.github/workflows/build.yaml

```diff
@@ -68,9 +68,9 @@ jobs:
             top-commit.txt
 
   release:
-    if: >
-      ${{ github.ref == 'refs/heads/timberland_1.0_final' &&
-          github.event_name == 'push' }}
+    if: |
+          github.ref == 'refs/heads/timberland_1.0_final' &&
```

Comment:
I believe the curly braces were wrong but the `>` could be kept as-is. See 4) in https://github.com/orgs/community/discussions/25641.

Anyway, if you tested it and found it works, fine with me.

Reply:
yup, seemed to work in my test branches at least as "skipped" and even below in the Pull check skipped it correctly this time.   People can still get to the output artifact from the build page too if they want it.